lib/ret_web/channels/hub_channel.ex

```diff
@@ -128,6 +139,11 @@ defmodule RetWeb.HubChannel do
 
       send(self(), {:begin_tracking, socket.assigns.session_id, hub.hub_sid})
 
+      # Send join push notification if this is the first joiner
+      if Presence.list(socket.topic) |> Enum.count() == 0 do
+        Task.start_link(fn -> hub |> Hub.send_push_messages_for_join() end)
```

Comment:
Shouldn't the push be sent on the second person joining (`Presence.list(socket.topic) |> Enum.count() == 1`)?

Reply:
Ah, nvm, I get it now, we only send push notifications when the subscriber has closed the tab.